### PR TITLE
0.1.1 Ingestion (Parquet, BigQuery)

### PR DESCRIPTION
**Pull request by tusharchou**
# Local Data Ingestion
> To use iceberg sqlite catalog to query iceberg tables locally

## Journey So Far
In the first release of local-data-platform 0.1.0 we had lauched helloworld. From that in the last sprint we have come far by implemting a google cloud service called BigQuery as source and writting data in a target iceberg table

## Issues To-do

- [ ] #5 Test 0.1.1 pre release
- [ ] #17 pypi release notes and read me
- [ ] #8 Demo NYC data ingestion

## Feature To Build

- [ ] Ability to query local lake house
- [ ] Ability to query bigQuery

### CODE REVIEW

File: local-data-platform/local_data_platform/etl.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+        **Json(
+            name='nyc_taxi',
+            path=os.getcwd()+'/real_world_use_cases/nyc_yellow_taxi_dataset/config.json',
+            format='JSON'
+        ).get()
+    )
+    print(f" config.metadata {config.metadata['source']}")
+    etl = PyArrow(config)
+    df = etl.extract()
+    etl.load()
```
Comment (tusharchou) [inline local-data-platform/local_data_platform/etl.py:20]: @mrutunjay-kinagi be amazed

File: local-data-platform/local_data_platform/etl.py
Comment (mrutunjay-kinagi) [inline local-data-platform/local_data_platform/etl.py:1]: Are we going to move this file to other package ?

**Review by mrutunjay-kinagi — APPROVED** [review]
LGTM !

**Review by abhishekkar0603 — APPROVED** [review]
Approving this PR for now, this practice will not be accepted in future

**Pull request merged**